components/Header.js

```diff
@@ -1,6 +1,70 @@
-import Img from "./Logo";
+import Logo from "./Logo";
 import Link from "./Link";
 import Burger from "./BgMenu";
+import styled from "styled-components";
+
+const ContainerMobile = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #f9a825;
+  height: 80px;
+
+  @media (min-width: 768px) {
+    display: none;
+  }
+`;
+
+const Container = styled.div`
+  display: none;
+
+  @media (min-width: 768px) {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #f9a825;
+    height: 80px;
+  }
+`;
+
+const LogoContainer = styled.div`
+  padding: 10px 30px;
+  width: 60px;
+  height: 60px;
+`;
+
+const NavBarMobile = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px 30px;
+`;
+
+const NavBar = styled.div`
+  padding-right: 30px;
+  font-weight: 1000;
+`;
+
+const MenuLink = styled.a`
+  color: #444141;
```

Comment:
One very useful thing about styled-components is the ability to use themes, so common styling variables can be store in one place. This makes it easier to update the variables if needed, (there's a principle for this, it's called DRY - don't repeat yourself)